### DISCUSSION TITLE
Congratulations to our CNCF Google Summer of Code 2023 Participants! 🎉

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#954] [Google Summer of Code 2023]
Hello everyone,

We are excited to announce that the CNCF has had 14 projects selected for the Google Summer of Code 2023 program! 🚀

We want to congratulate all the fantastic contributors and dedicated mentors who will be working together on various projects throughout the summer. This is an excellent opportunity to learn, grow, and contribute to the CNCF ecosystem. We look forward to seeing the progress and accomplishments of each project over the coming months.

Here are the selected projects, contributors, and assigned mentors for GSoC 2023:

| Project Title                                                                          | Contributor       | Assigned Mentor(s)                                                                                            |
| ---                                                                                    | ---               | ---                                                                                                           |
| Support OpenMetrics _created timestamp in Prometheus                                   | Arthur Sens       | Max Amin (@macxamin), Bartek Plotka (@bwplotka), Daniel Hrabovcak (@TheSpiritXIII)  |
| Add Support for SQL Database Storage in CurveBS                                        | Bokai Jia         | Xiaocui Li  (@ilixiaocui)                                                                                     |
| Enhancements for Dockerfiles                                                           | Darshan Kumar     | Natalie Arellano (@natalieparellano)                                                                          |
| Dataplane migration for Apache Kafka communications: Leveraging Java19’s Project Loom. | Debasish Biswas   | Ali Ok (@aliok), Christoph Stäbler (@creydr), Pierangelo Di Pilato (@pierDipi), Matthias Wessendorf (@matzew) |
| Kubebuilder phase 2 plugin                                                             | Eileen Yu         | Rashmi Gottipati (@rashmigottipati), Camila Macedo (@camilamacedo86), Bryce Palmer (@everettraven)            |
| Jaeger: Implement ClickHouse support                                                   | Ha Anh Vu         | Yuri Shkuro (@yurishkuro)                                                                                     |
| Conmon-rs support for podman                                                           | mgjm              | Peter Hunt (@haircommander), Matt Heon (@mheon)                                                               |
| Streamlined OpenFeature Enhancement Proposal Process                                   | none_24           | Michael Beemer (@beeme1mr),  David Hirsch (@DavidPHirsch)                                                     |
| Timeframe for Metrics                                                                  | Rakshit Gondwal   | Thomas Schuetz (@thschue), Florian Bacher (@bacherfl)                                                         |
| Falco: Web IDE for Security Rules with WebAssembly                                     | Rohith Raju       | Jason Dellaluce (@jasondellaluce)                                                                             |
| Kubectl plugin for Armada                                                              | Shivang Shandilya | Kevin Hannon (@kannon92), Geoff Wilson (@suprjinx)                                                            |
| Prototyping Support for Content Addressable Systems like IPFS in TUF                   | Shubham Nazare    | Aditya Sirish A Yelgundhalli (@adityasaky), John Ericson (@Ericson2314)                                       |
| IDE Plugins for editing vela applications                                              | Swastik Gour      | Da Yin (@Somefive)                                                                                            |
| GitHub Actions for KubeArmor                                                           | zhy76             | Barun Acharya (@daemon1024), Ankur Kothiwal (@Ankurk99), Anurag Kumar(@kranurag7)                             |

We encourage the CNCF community to welcome and support these contributors, especially during the first weeks. Let's help them feel excited and invested in our community early on.

Once again, congratulations to all participants, and let's make this summer a great experience for everyone involved!

**Comment by AmanSarraf** [comment]
Congratulations everyone 👏

**Comment by satyampsoni** [comment]
congratulations everyone!

**Comment by haanhvu** [comment]
Thank you @nate-double-u @aliok (and anyone else in the admin team) for your support. The application process went smooth thanks to your clear instructions and prompt responses!

**Comment by ilixiaocui** [comment]
<img width="651" alt="image" src="https://github.com/cncf/mentoring/assets/15689619/bf1dd068-78cf-4a6b-8b09-1b16329b37f8">

Hello @nate-double-u , I received an email from Google and I want to confirm whether the `Org Admins` here refers to CNCF.

**Reply by aliok** [reply]
hey @ilixiaocui Yes, it is CNCF

**Reply by ilixiaocui** [reply]
Thanks for the reply!

**Comment by mohdfarhannawaz** [comment]
Congratulations Everyone on this fantastic opportunity 🥳

QUESTION :
As a freshman who is interested in Cloud Native technologies and stuff and wants to contribute to open-source, I was curious how could a beginner like me could contribute to these fantastic projects and have a positive impact. If anyone can guide me toward how I can contribute as a beginner, the tech stacks that I should learn in order to be able to contribute, and the communities and projects that would be eager/happy to have me as a contributor?

My motivation behind this is to get started into these interesting technologies, contribute as much as I can and implement my learning by doing along the journey, get hands-on the real-world projects, grow in the community and gain real-world experience and interact with link-minded folks and give back to the community by helping others.
My Skillset = ( C/C++, OOPS, beginner-level Fronted development HTML, CSS, JS )

Thanks for your time and efforts in reading this  : )

**Reply by satyampsoni** [reply]
Good Question!
If you want to begin with contributing I would suggest you to find the projects around your skillset understand how Git and GitHub works. Learn about the important terminologies like commits, issues, pull requests, discussions, code of conduct and all and try contributing in those projects and in parallel start learning about cloud. Start with what is cloud computing you will find an overview of it on YouTube by David J Malan. Learn about what cloud native is and it's objectives  learn the tools and technologies related to it and try to find projects into this domain that interests you and look for some doc contribution and try to set up the project locally and learn how the project works and if you find any issues setting in up ask your questions in the community channels. If you find the solution that good if don't look for the things that was not available there that made you trouble setting up that project and raise an issue about it and solve that issue . Most of the project is very hard to set up. Make the process easy. Keep learning about it keep contributing around it. Hope this helps.

**Reply by aliok** [reply]
hey @mohdfarhannawaz 

In addition to @satyampsoni 's suggestions, we have a great guide that can answer all of your questions: https://contribute.cncf.io/contributors/getting-started/

**Reply by mohdfarhannawaz** [reply]
Thanks, @satyampsoni and @aliok  for your suggestions and time.  I would definitely check out the guide and follow the advice.
Thank you very much : )

**Comment by CodexRaunak** [comment]
hey, I want to contribute in the cloud native projects. Can somebody tell what are the current projects of CNCF where one can contribute to?
I am starting out my open source journey, it would be a lot of help if someone could point out in the right direction. I currently know Java, can I contribute with it or GO is required?
